CHANGES.md

```diff
@@ -116,6 +116,8 @@
 
 ## Bugfixes
 
+* Fix S3 copy for large objects ([BEAM-14011](https://issues.apache.org/jira/browse/BEAM-14011))
```

Comment:
This is for beam 2.37, which as I understand it is already cut/in progress.  So this will have to move to the 2.38 section

Reply:
moved.